manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -109,22 +101,18 @@ spec:
         severity: warning
     - alert: etcdBackendQuotaLowSpace
       annotations:
-        summary: etcd cluster database size exceeds the defined quota.
-        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the
-          defined quota on etcd instance {{ $labels.instance }}, please defrag or
-          increase the quota as the writes to etcd will be disabled when it is full.'
+        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the defined quota on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.'
```

Comment:
"... exceeds the defined quota..." is a bit premature.  I'd just say "... is {{ $value }}% of the defined quota..." or some such.